src/main/java/service/report/ReportServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package service.report;
+
+import dao.Storage;
+import model.FruitRecord;
+
+import java.util.List;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String FRUIT = "fruit";
+    private static final String QUANTITY = "quantity";
+    private static final String COMMA = ",";
+
+    @Override
+    public String getReport(List<FruitRecord> fruitRecords) {
```

Comment:
According to the checklist, we should avoid hardcoding values. The `getReport` method currently uses the `fruitRecords` parameter which is not being used inside the method. This parameter should be utilized to generate the report instead of using the static `Storage` directly.